passes/memory/memory_bram.cc

```diff
@@ -326,7 +328,18 @@ struct rules_t
 				data.or_next_if_better = true;
 				continue;
 			}
+	
+			if (GetSize(tokens) == 3 && tokens[0] == "!attribute") {
```

Comment:
This would mean you can't have a rule like so:
`attribute !ram_style ram_style=block`
which will allow either the omission of `ram_style`, or if it was set to the value `block`
Would this be a problem?